app/page.tsx

```diff
@@ -9,14 +9,39 @@ export default function Page() {
       <PageTitle>Department of Decentralization</PageTitle>
       <div className="prose max-w-none pb-8 pt-8 dark:prose-invert">
         <div>
-          The Department of Decentralization is a collective of people from various crypto,
-          decentralization and peer-to-peer communities in and around Berlin. The group assembled in
-          2018 to organize ETHBerlin and has been active since.
+          The Department of Decentralization is a collective of people from the cryptocurrency,
+          decentralization, and peer-to-peer community in Berlin. The group assembled in 2018 to
+          organize the{' '}
+          <a href="https://ethberlin.com" target="_blank" rel="noopener noreferrer">
+            ETHBerlin
+          </a>{' '}
+          hackathon and has been active since.
         </div>
         <div>
           We aim to be an agnostic vehicle to drive adoption, educate newcomers, and raise awareness
           of the challenges and benefits of decentralization and open-source software.
         </div>
+        <div>
+          All our events are free to attend and we try, whenever possible, to deliver a
+          distraction-free experience by not hosting sponsors, paid talks, or any other commercial
+          components. Our collective entirely runs on donations.
+        </div>
+        <div>
+          <img
+            alt="The DoD team at Protocol Berg v1 in 2023"
+            src="/static/images/team.jpg"
```

Comment:
```suggestion
            src="/static/images/team.jpg"
```
I would like this image more as it shows our team is bigger than just a handful of people.